com.github.coslyk.MoonPlayer.json

```
@@ -11,7 +11,7 @@
       "--device=all",
       "--share=network",
       "--socket=pulseaudio",
-      "--filesystem=host",
+      "--filesystem=home",
```

Why does the app need full home access, isn't portal enough?
```suggestion
      "--filesystem=xdg-videos",
      "--filesystem=xdg-download",
```

---

I assume that users may want to open the videos from other directories, such as from USB sticks.

So I should add these options, right?
```
"--filesystem=host:ro",
"--filesystem=xdg-videos",
"--filesystem=xdg-download",
```

---

Many thanks for your suggestion, I will change it.